static/app/views/insights/browser/webVitals/components/tables/pageSamplePerformanceTable.tsx

```diff
@@ -375,14 +377,16 @@ export function PageSamplePerformanceTable({transaction, search, limit = 9}: Pro
         <NoOverflow>
           <AlignCenter>
             {replayTarget &&
-              Object.keys(replayTarget).length > 0 &&
-              replayExists(row[key]) && (
-                <Tooltip title={t('View Replay')}>
-                  <LinkButton to={replayTarget} size="xs">
-                    <IconPlay size="xs" />
-                  </LinkButton>
-                </Tooltip>
-              )}
+            Object.keys(replayTarget).length > 0 &&
+            replayExists(row[key]) ? (
+              <Tooltip title={t('View Replay')}>
+                <LinkButton to={replayTarget} size="xs">
+                  <IconPlay size="xs" />
+                </LinkButton>
+              </Tooltip>
+            ) : (
+              <NoValue>{' \u2014 '}</NoValue>
```

Comment:
👍🏻 dump it into a constant though, somewhere in a module settings file, so all the strings always match in case they change